### PR TITLE
New package: iamxiaojianzheng.jd-gui version 1.6.8

### LINKED ISSUE #154838: [Package Request]: Add JD-GUI 1.6.8
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/iamxiaojianzheng/jd-gui/releases/tag/v1.6.8
Publisher: xiaojianzheng
Package Name: JD-GUI
Description: JD-GUI, a standalone graphical utility that displays Java sources from CLASS files.
Package Version: 1.6.8
Installer URL: https://github.com/iamxiaojianzheng/jd-gui/releases/download/v1.6.8/jd-gui-windows-1.6.8.zip

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #154838

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154862)

### CODE REVIEW

**Comment by Dragon1573** [comment]
This package requires a _Java Development Kit_ (version 1.8 or newer) to run properly. So I introduce `Microsoft.OpenJDK.21` here.

Other JDKs are accepted, so maybe `Dependencies` of this package should be adjust.

**Comment by Dragon1573** [comment]
Installation process is verified in Windows Sandbox, and proved to be valid.

Could moderators please provide the error logs? 🙏🏼

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-05-23 08:52:01.989 [CLI ] No suitable installer found for manifest Microsoft.OpenJDK.21 with version 21.0.3.9

(Automated response - build 881.)

File: manifests/i/iamxiaojianzheng/jd-gui/1.6.8/iamxiaojianzheng.jd-gui.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: iamxiaojianzheng.jd-gui
+PackageVersion: 1.6.8
+Scope: machine
```
Comment (Dragon1573) [inline manifests/i/iamxiaojianzheng/jd-gui/1.6.8/iamxiaojianzheng.jd-gui.installer.yaml:6]: `Scope` key has added to `installer.yaml`, but it still can't pass the pipeline. 😕

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-05-23 18:57:41.091 [CLI ] No suitable installer found for manifest Microsoft.OpenJDK.21 with version 21.0.3.9

(Automated response - build 881.)

**Comment by Dragon1573** [comment]
```powershell
.\Tools\SandboxTest.ps1 .\manifests\i\iamxiaojianzheng\jd-gui\1.6.8\ -WinGetOptions '--scope machine'
```

image
image

**Comment by Dragon1573** [comment]
Is this a blocking pull request because of #152555?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-05-28 15:43:33.159 [CLI ] No suitable installer found for manifest Microsoft.OpenJDK.21 with version 21.0.3.9

(Automated response - build 881.)

**Comment by Dragon1573** [comment]
> Automatic Validation ended with:
> 
> > 2024-05-28 15:43:33.159 [CLI ] No suitable installer found for manifest Microsoft.OpenJDK.21 with version 21.0.3.9
> 
> (Automated response - build 881.)

I'll try switching its dependency to another JDK/JRE distribution.

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**